src/test/js/shared/custom_steps.js

```diff
@@ -3,23 +3,57 @@ var TestData = require('../config/test_data');
 module.exports = function() {
   return actor({
 
-    lockAccount: function (email) {
+    lockAccount: function (email, serviceName) {
+        var registerPage = TestData.WEB_PUBLIC_URL + '/users/selfRegister?redirect_uri=https://www.autotest.com&client_id=' + serviceName;
+        var loginPage = TestData.WEB_PUBLIC_URL + '/login?redirect_uri=https://www.autotest.com&client_id=' + serviceName + 'state=';
```

Comment:
needs to be "&state=" ?

Reply:
Not sure how that got removed - added back